lib/algolia/client.rb

```diff
@@ -399,7 +399,7 @@ def thread_local_hosts(read)
       hosts = Thread.current[thread_hosts_key]
       thread_index_key = read ? "algolia_search_host_index_#{application_id}" : "algolia_host_index_#{application_id}"
       current_host = Thread.current[thread_index_key].to_i # `to_i` to ensure first call is 0
-      if current_host != 0 && hosts[current_host][:last_call].to_i < Time.now.to_i - 60
+      if current_host != 0 && hosts[0][:last_call].to_i > Time.now.to_i - 60
         # the current_host is not the first one and we've been using it for less than a minute; continue doing so
```

Comment:
Oh wow, the `<` is wrong, but the `current_host` is correct.

Scenario:
 - do a call
 - it targets the host `0` and works
 - do a call
 - it targets the host `0` and fails
 - we retry with host `1` and store `current_host = 1`, and works
 - do a call
 - we're on host `1`, and it has been less than a minute that we've been using the host `1`: aka `hosts[current_host][:last_call].to_i > Time.now.to_i - 60`
 - it targets host `1`, and works
 - wait 1 min
 - do a call
 - we're on host `1`, and it has been MORE than a minute that we've been using the host `1`
 - return the regular `hosts` array, targets host `0` and works